server/db/migrations/20190605110045-create-book.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable require-jsdoc */
+
+export function up(queryInterface, Sequelize) {
+  return queryInterface.createTable('Books', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: Sequelize.INTEGER
+    },
+    title: {
+      allowNull: false,
+      type: Sequelize.TEXT
+    },
+    description: {
+      allowNull: false,
+      type: Sequelize.TEXT
+    },
+    amount: {
+      allowNull: false,
+      type: Sequelize.NUMERIC
+    },
+    authorID: {
+      allowNull: false,
```

Comment:
What happens if the author is unknown?

Reply:
Yeah, that is true, do you want books without Author to be published?

Reply:
There's a possibility that might happen but then a relationship has to exist between the books and author